packages/block-library/src/image/block.json

```diff
@@ -102,6 +101,12 @@
 			}
 		}
 	},
+	"selectors": {
+		"border": ".wp-block-image img"
+	},
+	"editorSelectors": {
+		"border": ".wp-block-image img, .wp-block-image .wp-block-image__crop-area"
+	},
```

Comment:
Is there a need to repeat the same selector (`.wp-block-image img`) in `selectors` and `editorSelector`? Does it mean that when defined `editorSelectors` need to repeat all entries from `selectors`? I didn't get this impression when reading the documentation.

Reply:
The choice here was to override the selector in the editor rather than merge selectors strings. 

This allows the use of completely different editor selectors if the need arises due to different markup in the editor. The downside of that freedom was some extra verbosity. The Image block was one such case where we wanted the original selector as well as an additional one in the editor instead of a replacement selector. A possible use case where we want a completely different selector in the editor might be one of the blocks using `ServerSideRender`.

You can find how the editor and normal selectors are merged [here](https://github.com/WordPress/gutenberg/pull/46496/files#diff-b03597cc3da199e5aa5a94950e8241135904853e7c3b82d758e42744878afae7R3460-R3462) along with some [related discussion](https://github.com/WordPress/gutenberg/pull/46496#discussion_r1139625469).